package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@yext/answers-core",
-  "version": "1.4.0",
+  "version": "1.4.0-beta.0",
```

Comment:
Would it be more accurate to have 1.5.0-beta.0, since 1.4.0 is already released?

Reply:
right